pkg/innerring/internal/blockchain/blockchain.go

```diff
@@ -91,6 +91,42 @@ type PingConfig struct {
 	Timeout time.Duration
 }
 
+// RPCConfig configures RPC serving.
+type RPCConfig struct {
+	// Network addresses to listen Neo RPC on. Each element must be a valid TCP
+	// address in 'host:port' format.
+	//
+	// Optional: by default, Neo RPC is not served.
```

Comment:
can we provide TLS interface only? If no, then correct, otherwise
```suggestion
	// Optional: by default, insecure Neo RPC is not served.
```

Reply:
IMO, it's better to have an option here and to keep insecure connections availability.

Reply:
> can we provide TLS interface only?

i think yes, why not? (in fact, i expect it to be either TLS only or non-TLS only)

Reply:
Can be any way: TLS/non-TLS/both.

Reply:
then my change suggestion is valid, Neo RPC can be served with TLS even if `Addresses` is empty

Reply:
fixed